test/functional/data/invalid_txs.py

```diff
@@ -124,6 +126,22 @@ def get_tx(self):
         assert MIN_STANDARD_TX_NONWITNESS_SIZE - 1 == 64
         return tx
 
+# reject a transaction that contains a witness
+# but doesn't spend a segwit output
+class ExtraWitness(BadTxTemplate):
+    expect_disconnect = False
+    valid_in_block = False
```

Comment:
these two lines can be dropped; `expect_disconnect` was removed in #33050 (commit 876dbdfb4702410dfd4037614dc9298a0c09c63e) and `valid_in_block` is set to False by default anyways
```suggestion
```
other than that, lgtm

Reply:
Done in a7c96f8